mteb/models/nomic_models.py

```diff
@@ -16,6 +17,9 @@
 
 logger = logging.getLogger(__name__)
 
+MODERN_BERT_TRANSFORMERS_MIN_VERSION = (4, 48, 0)
+CURRENT_TRANSFORMERS_VERSION = tuple(map(int, transformers.__version__.split(".")[:3]))
+
```

Comment:
Could use packaging.verison (combined with below):
```suggestion
from packaging import version

MODERN_BERT_TRANSFORMERS_MIN_VERSION = "4.48.0"
```